recipe/meta.yaml

```diff
@@ -1,25 +1,22 @@
-{% set name = "packaging" %}
```

Comment:
Any reason why you replaced those Jinja directives by using inline names?

Reply:
Too much jinja makes for an unreadable recipe :wink:

Reply:
Besides, the only jinja variable that will be used when updating a recipe is `version`.

Reply:
I see, those Jinja variables were useful to copy recipes from one package to another, but now that `conda skeleton` generates them they are not as useful.

I would leave `version` and `hash` though because they are the ones changed when updating a recipe, but that can be done the next time we make a new release. 👍

Reply:
I see, those Jinja variables were useful to copy recipes from one package to another, but now that `conda skeleton` generates them they are not as useful.

Even then I disagree that the benefits were worth the complexity. I like a readable recipe :smile: 

> I would leave `version` and `hash` though because they are the ones changed when updating a recipe, but that can be done the next time we make a new release. 👍 

Version yes, `hash`... Not so sure. We are in the process of automating that and maybe having it in its proper field will come in handy when sharing recipes with anaconda aggregate.

Reply:
You mean https://github.com/regro/cf-graph? Well OK, thanks for the answer!